anndata/core/anndata.py

```diff
@@ -1,24 +1,25 @@
 """Main class and helper functions.
 """
-from enum import Enum
 from collections import OrderedDict
 from collections.abc import MutableMapping
+from copy import deepcopy
+from enum import Enum
 from functools import reduce
 from pathlib import Path
-from typing import Any, Union, Optional
-from typing import Iterable, Sized, Sequence, Mapping
-from typing import Tuple, List
-from copy import deepcopy
+from typing import (
+    Any, Union, Optional,
+    Iterable, Sequence, Mapping, Tuple, List
+)
```

Comment:
This was deliberate, but I should have added comments:

```py
from typing import Any, Union, Optional                # Meta
from typing import Iterable, Sized, Sequence, Mapping  # Abstract collections
from typing import Tuple, List                         # Concrete types
```

And on separate lines because that’ll survive black formatting once we switch to it.

Reply:
This is a little nitpicky on my part, but aren't the `Tuple` and `List` types generic, not concrete?

Reply:
Sure, I meant “concrete” in the sense of “non-abstract”. Is there a better word?